microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -33,7 +29,7 @@ Follow the major Step sections in sequence, below.
 <!-- ====================================================================== -->
 ## Step 1 - Download a preview channel of Microsoft Edge
 
-For full API compatibility of the **Microsoft.UI.Xaml** (WinUI 2) package and WebView2, this tutorial uses a _prerelease_ version of WebView2 SDK, together with a preview channel of Microsoft Edge.  (This tutorial doesn't use the other supported combination, which is a _release_ version of WebView2 SDK + the WebView2 Runtime.)
+For full API compatibility of the **Microsoft.UI.Xaml** (WinUI 2) package and WebView2, this tutorial uses a _prerelease_<!-- todo: update re "prerelease"? --> version of WebView2 SDK, together with a preview channel of Microsoft Edge.  (This tutorial doesn't use the other supported combination, which is a _release_ version of WebView2 SDK + the WebView2 Runtime.)
```

Comment:
@jasonstephen15 pls do "Find" on "prerelease" in this Files Changed page, and maybe in the rendered WinUI 2 article.  

Does this getstart require prerelease WebView2 SDK?  
Does this getstart require prerelease WinUI 2 SDK?

Reply:
ans:
no
no
Update screenshots.

Reply:
planned.  entering fresh cmt